apps/passport-client/src/dispatch.ts

```diff
@@ -680,6 +680,7 @@ async function resetPassport(state: AppState, update: ZuUpdate): Promise<void> {
   // Clear in-memory state
   update({
     self: undefined,
+    loggingOut: true,
```

Comment:
questions moreso for checking rather than pointing out actual issues

- when is this set to false?
- is this resilient to an error in `notifyLogoutToOtherTabs()` or otherwise that could cause this to be an infinite spinner?

Reply:
it's never set to false, it starts off as `undefined` and is only set to `true` on the tab where the user clicks 'log out'

> is this resilient to an error in notifyLogoutToOtherTabs() or otherwise that could cause this to be an infinite spinner?

I could reorder the code here to make this not an issue.

Reply:
sg